editor.planx.uk/src/lib/applications.ts

```diff
@@ -0,0 +1,49 @@
+import gql from "graphql-tag";
+import { client } from "lib/graphql";
+
+export type ApplicationData = {
+  session_id: string;
+  submitted_at: Date | string;
+  user_invited_to_pay: boolean;
+  payment_status: string;
+  amount: number;
+  payment_date: Date | string;
+  sent_to_email: boolean;
+  sent_to_bops: boolean;
+  sent_to_uniform: boolean;
+};
+
+export async function fetchSubmittedApplications(
```

Comment:
nit: I personally think this function & query feel a bit out of place in `/lib` and should be co-located in the index.tsx component file?

Reply:
That's fair enough, I fancies abstracting it out a bit like `model.ts` type files or `/lib/feedback` although this is very weakly held and I'm happy to update it 👍